core/src/main/java/cz/xtf/core/openshift/OpenShift.java

```diff
@@ -277,15 +279,20 @@ public KubernetesList createResources(List<HasMetadata> resources) {
     }
 
     public KubernetesList createResources(KubernetesList resources) {
-        return lists().create(resources);
+        return (KubernetesList) resourceList(resources.getItems()).create();
     }
 
     public boolean deleteResources(KubernetesList resources) {
-        return lists().delete(resources);
+        if (resourceList(resources.getItems()).delete() != null) {
```

Comment:
I think that `resourceList(resources.getItems()).delete()` will always be non null. I think we can compare size of original list with delete list 

```
return resources.getItems().size() == resourceList(resources.getItems()).delete().size(); 
```

Reply:
I think we can assume it would be just one resource, so either 
```
imageStreams().resource(imageStream).delete().size() == 1
```
or
```
!imageStreams().resource(imageStream).delete().size().isEmpty()
```